### ISSUE TITLE
[Failing Test]: 404 opensource.org/licenses cause java container build fail

### ISSUE THREAD

**Issue by Abacn** [apache/beam#30377]
### What happened?

e.g. https://opensource.org/licenses/cpl1.0.txt no longer exist

### Issue Failure

Failure: Test is continually failing

### Issue Priority

Priority: 0 (outage / failing precommit test impacting development)

### Issue Components

- [ ] Component: Python SDK
- [ ] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [ ] Component: Beam YAML
- [ ] Component: Beam examples
- [ ] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

**Comment by Abacn** [comment]
looks like it is resolved. However, the website is now an html: e.g. https://opensource.org/license/cpl1.0.txt

**Comment by kennknowles** [comment]
Maybe the best way to do this in the future (not sure how much work) is to run the same script to pull licenses but then check them in to the repository.

**Comment by kennknowles** [comment]
I think P1 if it is no longer failing?

**Comment by Abacn** [comment]
Yeah P2 suffices. Possible follow up is to switch to use a more reliable site (and plain text license)
